core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -1071,4 +1096,47 @@ class TestSuiteWriter {
             .toList()
     }
 
+    private fun getDtos(): List<DtoClass> {
+        val restSampler = sampler as AbstractRestSampler
+        val result = mutableListOf<DtoClass>()
+        restSampler.getActionDefinitions().forEach { action ->
+            action.getViewOfChildren().forEach { child ->
+                if (child is BodyParam) {
+                    val primaryGene = child.primaryGene()
+                    // TODO: Payloads could also be json arrays, analyze ArrayGene
+                    if (primaryGene is ObjectGene) {
```

Comment:
this could be inside an `OptionalGene`, or other kinds of wrapper genes. might want to get reference to the `ObjectGene` (if any) using `primaryGene.getWrappedGene`